### PR TITLE
Add tutorial - Securing a static website hosted on Google Cloud Storage 

### PR DESCRIPTION
**Pull request by xiangshen-dk**
This tutorial describes a solution to protect static assets on GCS based on the GCLB Serverless Network Endpoint Groups (NEGs).

Based on these docs: 
go/secure-static-gcs-site
go/secure-static-gcs-site-tutorial

### CODE REVIEW

**Comment by ToddKopriva** [comment]
@ptone, I added you as a reviewer after asking Xiang, since I see that you are a reviewer on the internal draft.

I'll do the editorial and production review after the technical review is done.

File: tutorials/securing-gcs-static-website/index.md [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+1. The login app performs authentication. If the authentication is successful, the login app [generates a signed cookie ](https://cloud.google.com/cdn/docs/using-signed-cookies)and sends it back to the client with an HTTP redirect. The cookie works because the login app and the bucket backends are behind the same load balancer. Therefore, they are considered as **the [same origin](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy) (Important!)**. Even if a user can access the default Cloud Run endpoint directly and log in from there, she still doesn't have access to the CDN since the cookie is not from the same origin.
+
+1. The CDN [signed cookie is verified by Cloud CDN](https://cloud.google.com/cdn/docs/using-signed-cookies), providing access to the static assets. We can specify an expiration time for the cookie. Once it's expired, modern browsers either delete the cookie or stop sending it. Cloud CDN will also reject expired cookies.
+
+1. Permit Cloud CDN to read the objects in our private bucket by adding the Cloud CDN service account to Cloud Storage's ACLs.
+
+## Objectives 
+
+-  Learn to build and deploy a static website to a Cloud Storage bucket.
```
Comment (ptone) [inline tutorials/securing-gcs-static-website/index.md:27]: IMO remove the repetitive "Learn to" from these, as at the end of the tutorial, you've actually done them, not just learned

File: tutorials/securing-gcs-static-website/index.md [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+
+    ```bash
+    sed -i -e "s/<DNS_NAME>/$DNS_NAME/" web-map-http.yaml
+    sed -i -e "s/<PROJECT_ID>/$PROJECT_ID/" web-map-http.yaml
+    sed -i -e "s/<LOGIN_BACKEND_SVC_NAME>/$LOGIN_BACKEND_SVC_NAME/" web-map-http.yaml
+
+    gcloud compute url-maps import web-map-http --source web-map-http.yaml --global
+    ```
+
+    In this step, we updated the values in the template URL map file and imported it. An example of the final configuration looks like the following:
```
Comment (ptone) [inline tutorials/securing-gcs-static-website/index.md:309]: This IS the final config, not really an example of

File: tutorials/securing-gcs-static-website/index.md [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+However, it's still challenging to provide authentication and authorization for a static website on GCP **in a serverless fashion**.  This tutorial describes a solution to protect static assets on GCS based on the GCLB [Serverless Network Endpoint Groups](https://cloud.google.com/load-balancing/docs/negs/setting-up-serverless-negs) (NEGs).  
+Below is the high-level architecture of this solution:
+
+![gcs-static-arch](static-website/src/assets/arch-gcs-site.png)
+
+The architecture above incorporates the following key features:
+
+1. Create a load balancer to use a custom domain with a managed SSL certificate. When a request is received, we use [routing rules](https://cloud.google.com/load-balancing/docs/https/setting-up-query-and-header-routing#http-header-based-routing) to check whether [a signed cookie for Cloud CDN](https://cloud.google.com/cdn/docs/private-content#signed_cookies) is in the request header. If the cookie doesn't exist, redirect the request to a login app hosted on Cloud Run. If there is a signed cookie, send the request to the Cloud CDN of the backend bucket.
```
Comment (ptone) [inline tutorials/securing-gcs-static-website/index.md:17]: Keep these as nouns - remove the "create" verb ... "A load balancer using a custom domain..."

File: tutorials/securing-gcs-static-website/index.md (hunk 2 above)
Comment (xiangshen-dk) [reply tutorials/securing-gcs-static-website/index.md]: removed the word 'example'

File: tutorials/securing-gcs-static-website/index.md (hunk 3 above)
Comment (xiangshen-dk) [reply tutorials/securing-gcs-static-website/index.md]: Updated as suggested.

File: tutorials/securing-gcs-static-website/index.md [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+
+For this tutorial, you need a Google Cloud [project](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects). You can create a 
+new one, or select a project you already created.
+
+1.  [Select or create a Google Cloud project.](https://cloud.console.google.com/projectselector2/home/dashboard)
+
+1.  [Enable billing for your project.](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+
+1.  [Enable the Compute Engine, Cloud Run, Cloud Secret Manager, and Cloud DNS APIs.](https://console.cloud.google.com/flows/enableapi?apiid=dataproc,storage_component)
```
Comment (ToddKopriva) [inline tutorials/securing-gcs-static-website/index.md:72]: The APIs listed in the text don't match the APIs passed in the URL parameter.

**Review by ToddKopriva — APPROVED** [review]

**Comment by ToddKopriva** [comment]
https://cloud.google.com/community/tutorials/securing-gcs-static-website

**Pull request merged**